src/Writer.jl

```diff
@@ -1,5 +1,6 @@
 module Writer
 
+using Compat
```

Comment:
I think you also need a `using Compat.Dates`

Reply:
@nalimilan is the `using Compat.Dates` required as well?

Reply:
Yes, you need to replace `Base.Dates` with `Compat.Dates` everywhere it appeared. So I now realize https://github.com/JuliaData/Missings.jl/pull/56 didn't change anything to the problem.

Reply:
Regarding the case at hand, you just need to replace `Dates` with `Compat.Dates` on this line: https://github.com/JuliaIO/JSON.jl/blob/b4a4e338029bb999589dfabc6d90f5ae2fdd08d8/src/Writer.jl#L52

Reply:
Let us check change it and check Travis. After the manual fix both packages compiled so the CI should be cleared.